ch32fun/ch57xhw.h

```diff
@@ -1628,6 +1633,172 @@ typedef enum
 #define R8_UEP7_T_LEN       (*((vu8*)0x4000806C))  // endpoint 7 transmittal length
 #define R8_UEP7_CTRL        (*((vu8*)0x4000806E))  // endpoint 7 control
 
+//some probabily works for ch571/3 too, but not tested
+//so only for ch570/2 for now.
+#if( MCU_PACKAGE == 2 || MCU_PACKAGE == 0) // CH570/2
+
+#define RTC_MAX_COUNT       0xA8BFFFFF
+
+// LSI frequency is actually unknown and can't be tuned. 
+// It can be anywhere between 24 kHz to 42 kHz. If percise timing is needed,
+// the user should calibrate it with R8_OSC_CAL_CTRL and
+// R16_OSC_CAL_CNT.
+#define RTC_FREQ            32000 
+
+// The datasheet of ch570/2 refer to some registers differently.
+// They should be defined in case the user is referring to the datasheet.
```

Comment:
This is a good point, there are several examples where 570/2 has different names for the same thing compared to ch571/3. I personally think there should go side by side where the pointer is defined, but I'll let @cnlohr decide on where (and if?) these should be defined.